projects/canopy/src/lib/modal/modal-header/modal-header.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../../styles/mixins';
+
+.lg-modal-header {
+  border-bottom: var(--border-width) solid var(--color-platinum);
+  display: flex;
+  justify-content: space-between;
+  padding: var(--space-sm);
+
+  @include lg-breakpoint(md) {
+    padding: var(--space-md);
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
```

Comment:
Just a thought - I'm starting to wonder if having that class input on `lg-header` was actually quite useful 🤔 , as quite often in Canopy and when using Canopy, I find that I need override the heading styles for some reason. I can't remember the rationale in removing it now 😄

Reply:
I think it was because it was allowing people to add their own classes 🤔  maybe we should add a hardcoded class like we do for all the other components instead. It will make these type of things a lot nicer to target.